Modify documentation (#399)
ansys/pyansys-dev-guide: doc/source/how-to/setting-up.rst

        .. note::

            It is likely that your Linux/UNIX distribution already comes with some
            version of Python installed. For more information, see :ref:`Verify python version`.

> Verify python version

Reviewer: ```suggestion
Verify Python version
```

Create
~~~~~~

Usually, virtual environments are named ``venv`` or ``.venv``.
You can create a virtual environment named ``<venv>`` with:

.. code-block:: text

    python -m venv <venv>

Activate
~~~~~~~~

> You would activate the preceding virtual environment with:

Reviewer 2: ```suggestion
You would activate the preceding virtual environment with the command for your OS:
```

                    <venv>\Scripts\Activate.ps1

    .. tab-item:: macOS/Linux/UNIX

        .. code-block:: text

            source <venv>/bin/activate

Deactivate
~~~~~~~~~~

> You could deactivate a virtual environment with:

Reviewer 2: ```suggestion
You could deactivate a virtual environment with the command for your OS:
```

        1. Check the `latest stable Git version for Linux/UNIX <https://git-scm.com/download/linux>`_.
        2. Run the installation command for your package manager.

> Verify git version

Reviewer 2: ```suggestion
Verify Git version
```

~~~~~~~~~~~~~~~~~~

Once your installation process is complete, verify your Git installation with:

.. code-block:: text

    git --version

Usage
~~~~~

> If you're new to Git, refer to the `Git Reference Manual <https://git-scm.com/doc>`_
> for comprehensive usage details.

Reviewer 2: ```suggestion
If you're new to Git, see the `Git Reference Manual <https://git-scm.com/doc>`_
for comprehensive usage information.
```

For an understanding of Git workflows and branching strategies, 

> check out the `Learning Git branching <https://learngitbranching.js.org/>`_ tutorial.

Reviewer 2: ```suggestion
see the `Learning Git branching <https://learngitbranching.js.org/>`_ tutorial.
```

> If you're unfamiliar with GitHub, explore the 

Reviewer 2: ```suggestion
If you're unfamiliar with GitHub, see the 
```
